bundles/org.openhab.binding.iammeter/pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
version section changed.

Reply:
I found it can't pass the Travis test if fill it '2.5.3-SNAPSHOT' here,only '2.5.7-SNAPSHOT' can pass Travis test.
so I changed it to 2.5.7-SNAPSHOT,hope it doesn't affect other parts.

Reply:
That's the way it should be. 2.5.7-SNAPSHOT is fine.